samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -455,8 +454,6 @@ object SamzaContainer extends Logging {
         loggedStorageBaseDir = defaultStoreBaseDir
       }
 
-      storeWatchPaths.add(loggedStorageBaseDir.toPath)
```

Comment:
Just for my understanding: previous it is counting all the task stores for an application on a node?